apps/dg/components/case_card/case_card.js

```diff
@@ -385,20 +385,51 @@ DG.React.ready(function () {
             }.bind(this));
           },
 
+          /**
+           * -------------------Get first child case of selected case-----------------
+           */
+          getSelectedCaseFirstChildCaseID: function (iCollection) {
+              var tSelectedCaseFirstChildCaseID,
+                  tContext = this.props.context;
+              var tSelectedCase = tContext.getCollectionByID(iCollection.get('id')).getPath('casesController.selection').toArray();
+              if (tSelectedCase.length) {
+                tSelectedCaseFirstChildCaseID = tSelectedCase[0].children[0].id;
```

Comment:
In SproutCore parlance, this should probably be `tSelectedCase[0].get('children')[0].id`.